apps/sms/js/thread_ui.js

```diff
@@ -1387,26 +1389,38 @@ var ThreadUI = {
    * markup to support bidirectional content, but other panels still use it with
    * mozL10n.setAttributes as it would contain only localizable text. We should
    * get rid of this method once bug 961572 and bug 1011085 are landed.
-   * @param {string|{ id: string, args: Object }} content Should be either safe
-   * HTML string or object with l10nId and l10nArgs.
+   * @param {string|{ raw: string }|{ html: string }|
+   * {id: string, args: Object }} contentL10n Should be either safe HTML string
+   * or l10n properties or raw string.
    * @public
    */
-  setHeaderContent: function thui_setHeaderContent(content) {
-    if (typeof content === 'string') {
-      this.headerText.removeAttribute('data-l10n-id');
-      this.headerText.removeAttribute('data-l10n-args');
-
-      this.headerText.innerHTML = content;
-    } else {
+  setHeaderContent: function thui_setHeaderContent(contentL10n) {
+    if (typeof contentL10n === 'string') {
```

Comment:
Sure, I can join these two cases.  I remember some time ago someone from l10n team said that its better to use ```setAttributes``` only when we have l10nArgs, but anyway I think like you here :)